src/main/java/com/ibm/mobilefirstplatform/serversdk/java/push/PushNotifications.java

```diff
@@ -187,7 +193,16 @@ public static void send(Notification notification, PushNotificationsResponseList
 			return;
 		}
 
-		CloseableHttpClient httpClient = HttpClients.createDefault();
+		CloseableHttpClient httpClient = null;
+		SSLContext sslContext;
+		try {
+			sslContext = SSLContexts.custom().useProtocol(TLS_VERSION).build();
+			httpClient = HttpClients.custom().setSSLContext(sslContext).build();
+		} catch (KeyManagementException exception) {
```

Comment:
Can you point me to which code in the above throws these two exceptions?  I dont think KeyManagementException is thrown